src/models/videojs.js

```diff
@@ -109,7 +96,7 @@ const updatePosterUrl = (posterImage, width) => {
 
 class VideoJsPlayer extends Video {
 	constructor(el, opts) {
-		advertising = opts && opts['advertising'] ? true : false;
```

Comment:
No proper reason, I just changed it because it was shorter and I liked it more. I can change it back. After the refactor, there will probably be changes in the module instantiation process though that will require changes across the different apps